backup-index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quizzzy</title>
+    <link rel="stylesheet" href="styles/index.css" />
+    <link href="src/fontawesome/css/all.css" rel="stylesheet" />
+  </head>
+  <body class="app-grid">
+    <header class="app-grid__header centered-grid">
```

Comment:
Your header could be considered as a new starting element within BEM.
```suggestion
    <header class="header centered-grid">
```